### PR TITLE
glsl: Replace todo!() with a NotImplemented error

### PR DESCRIPTION
**Pull request by magcius**
So we can get a source location rather than a panic.

### CODE REVIEW

**Review by kvark — APPROVED** [review]
Nice!

**Comment by kvark** [comment]
Thank you for the first contribution!
Clippy isn't happy

File: src/front/glsl/parser.rs [hunk 1]
```diff
@@ -797,7 +797,10 @@ impl<'source, 'program, 'options> Parser<'source, 'program, 'options> {
                             //TODO: declaration
                             // type_qualifier IDENTIFIER SEMICOLON
                             // type_qualifier IDENTIFIER identifier_list SEMICOLON
-                            todo!()
+                            Err(ErrorKind::NotImplemented(
+                                token.meta,
+                                "type qualifier",
```
Comment (JCapucho) [inline src/front/glsl/parser.rs:802]: This message is quite ambiguous can't we name something like `variable modifiers`?

**Review by JCapucho — APPROVED** [review]
LGTM

**Pull request merged**